flow/project.py

```diff
@@ -128,7 +128,7 @@ def false(cls, key):
 
     @classmethod
     @deprecated(
-        deprecated_in="0.9", removed_in="1.0",
+        deprecated_in="0.9", removed_in="0.11",
```

Comment:
Should we add `, current_version=__version__` for consistency?

Reply:
Good find! I hadn't noticed that `current_version` was missing. I fixed it.